src/psyclone/tests/gocean1p0_transformations_test.py

```diff
@@ -1655,111 +1617,6 @@ def test_acc_data_grid_copyin(tmpdir):
     assert GOcean1p0Build(tmpdir).code_compiles(psy)
 
 
-def test_acc_rscalar_update(tmpdir):
```

Comment:
Again, we need to check that we can safely do away with this functionality. Maybe the fact that we now always do an `enter data` (i.e. there's no `if(first_time)...`) *and* include scalars in it covers us - I think it should.

Reply:
As mentioned before, I'll test this.

Reply:
The code has been tested using example gocean/eg2 on glados and correct results obtained with scalars (both constant values and changing values). 55.0 is the magic number (1+2+....+10). Thanks to @arporter for helping with this. Running this example did demonstrate a separate problem which is captured in issue #491.